contrib/pkg/createcluster/create.go

```diff
@@ -392,6 +402,42 @@ func (o *Options) Validate(cmd *cobra.Command) error {
 			return fmt.Errorf("unable to parse key=value label: %s", ls)
 		}
 	}
+
+	if len(o.HttpProxy) > 0 || len(o.HttpsProxy) > 0 {
+		if o.HttpProxy == "" && o.HttpsProxy == "" {
```

Comment:
This doesn't look quite right, I can't see how line 407 could ever be true given line 406.